Misc/NEWS.d/next/Library/2024-07-06-12-37-10.gh-issue-121423.vnxrl4.rst

```diff
@@ -0,0 +1 @@
+Speed up socket.errorTab and lazy import selectors
```

Comment:
```suggestion
Improve import time of `mod`:`socket` by writing `socket.errorTab` as a constant and lazy import of `mod`:`selectors`.
```

Reply:
@eendebakpt It's <code>:mod:\`socket\`</code>, not <code>\`mode\`:\`socket\`</code> and I would suggest having <code>:data:\`socket.errorTab\`</code>, or <code>:data:\`!socket.errorTab\`</code> depending on whether the data is documented or not.